lib/karma-electron-launcher.js

```diff
@@ -1,11 +1,22 @@
 // Load in our dependencies
 // DEV: We use `require`/`assert` instead of `peerDependencies` to make floating dependencies easier
 var electronPrebuilt;
-try {
+
+function moduleExists(moduleName) {
+  try {
+    return require.resolve(moduleName);
+  } catch (err) {
+    return false;
+  }
+}
+
+if (moduleExists('electron-prebuilt')) {
   electronPrebuilt = require('electron-prebuilt');
-} catch (err) {
-  console.error('Expected `electron-prebuilt` to be installed but it was not. Please install it.');
-  throw err;
+  console.warn('WARN: Consider changing from `electron-prebuilt` to `electron` has it will be deprecated soon.');
+} else if (moduleExists('electron')) {
+  electronPrebuilt = require('electron');
+} else {
+  throw new Error('Expected an electron package to be installed but none was found. Please install `electron`.');
```

Comment:
I wanted to mention the two packages name, but since the lint limit us to a number of characters per line, it was too long.